PostHog/Replay/UIApplicationTracker.swift

```diff
@@ -44,7 +44,7 @@
             guard event.type == .touches else {
                 return
             }
-            guard let window = PostHogReplayIntegration.getCurrentWindow() else {
+            guard let window = UIApplication.getCurrentWindow() else {
```

Comment:
This likely needs `checkForegrounded: true` since its swizzled in the main thread during touches, previously it did and now it does not, any reason why not?
Touches and screenshots need to be from the very same screen otherwise click on screen A might be appearing on screen B and vice versa I believe (until we have proper support for multi windows)